include/spdlog/sinks/qt_sinks.h

```diff
@@ -162,7 +162,7 @@ class qt_color_sink : public base_sink<Mutex>
 
         const string_view_t str = string_view_t(formatted.data(), formatted.size());
         // apply the color to the color range in the formatted message.
-        auto payload = QString::fromLatin1(str.data(), static_cast<int>(str.size()));
+        auto payload = QString::fromUtf8(str.data(), static_cast<int>(str.size()));
```

Comment:
The problem with utf8 is that the color position might be be wrong and needs to get calculated precisely somehow.